Packages/com.chisel.editor/Chisel/Editor/Editor/Scene/Handles/ExtrustionHandle.cs

```diff
@@ -10,125 +10,132 @@
 
 namespace Chisel.Editors
 {
-	public enum ExtrusionState
-	{
-		None,
-		Cancel,
-		Commit,
-		Modified
-	}
+    public enum ExtrusionState
+    {
+        None,
+        Cancel,
+        Commit,
+        Modified
+    }
 
-	public static class ExtrusionHandle
-	{
-		internal static int s_HeightSizingHash = "HeightSizingHash".GetHashCode();
-		public static ExtrusionState DoHandle(Rect dragArea, ref Vector3 position, Axis axis, UnitySceneExtensions.SceneHandles.CapFunction capFunction = null)
-		{
-			var id = GUIUtility.GetControlID(s_HeightSizingHash, FocusType.Keyboard);
-			return ExtrusionHandle.Do(id, dragArea, ref position, axis, capFunction);
-		}
+    public static class ExtrusionHandle
+    {
+        internal static int s_HeightSizingHash = "HeightSizingHash".GetHashCode();
+        public static ExtrusionState DoHandle(Rect dragArea, ref Vector3 position, Axis axis, UnitySceneExtensions.SceneHandles.CapFunction capFunction = null, float? snappingSteps = null)
```

Comment:
Before when you sized a spiral stairs it would size the stairs in grid steps in the upwards direction.
But it makes more sense to make it possible to size upwards in step sizes, since otherwise you'd end up with missing or weirdly sized steps.